apiserver/facades/client/charms/service.go

```diff
@@ -19,6 +20,12 @@ type ModelConfigService interface {
 	ModelConfig(context.Context) (*config.Config, error)
 }
 
+// ModelService provides access to model service.
```

Comment:
We should be consistent and use the same comment as elsewhere
`// ModelService provides a subset of the model domain service methods.`